2022-06-10 (#159)
rustacean-station/rustacean-station.org: _episodes/interviews/2022-06-10-rebecca-rumbul.md

---
title: "Rust Foundation with Rebecca Rumbul"
date: 2022-06-10T15:30:00Z
file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e073-rebecca-rumbul.mp3
duration: "56:47"
length: "55601613"
#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
---

> Allen Wyma talks with [Rebecca Rumbul](https://twitter.com/rebeccarumbul), Executive Director & CEO at [Rust Foundation](https://foundation.rust-lang.org/). The Rust Foundation is an independent non-profit organization to steward the Rust programming language and ecosystem, with a unique focus on supporting the set of maintainers that govern and develop the project.

Reviewer: ```suggestion
Allen Wyma talks with [Rebecca Rumbul](https://twitter.com/rebeccarumbul), Executive Director and CEO at [Rust Foundation](https://foundation.rust-lang.org/). The Rust Foundation is an independent non-profit organization to steward the Rust programming language and ecosystem, with a unique focus on supporting the set of maintainers that govern and develop the project.
```

- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
- Github: [@rustacean-station](https://github.com/rustacean-station/)
- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)

## Timestamps 
- [@1:00] - Rebecca's [Bio](https://twitter.com/rebeccarumbul)
- [@2:36] - [The Rust Foundation](https://foundation.rust-lang.org/)
- [@7:27] - How the Rust Foundation deals with legal work
- [@9:26] - How the Rust Foundation helps all contributors
- [@12:47] - Scoring matrix to measure the value
- [@15:20] - [DevX Initiative](https://devxinitiative.org/) & [Ernest Kissiedu](https://github.com/ernestkissiedu)
- [@17:14] - Competing in funding projects
- [@20:29] - Applying for a [membership](https://foundation.rust-lang.org/members/) in The Rust Foundation
- [@23:25] - Company membership benefits
- [@28:34] - The Rust Foundation can potentially connect people and projects
- [@31:08] - Board member [Nell Shamrell-Harrington](https://twitter.com/nellshamrell) & The Rust Foundation sponsoring
- [@35:00] - Rebecca on making tough decisions
- [@36:46] - Nell's weekly newsletter

> - [@40:20] - Why is it that would make a company pay for a Platinum membership?

Reviewer: ```suggestion
- [@40:20] - What makes a company pay for a Platinum membership?
```

> - [@44:21] - Rebecca's [Background](https://www.linkedin.com/in/rebecca-rumbul-96a5441a/)

Reviewer: ```suggestion
- [@44:21] - Rebecca's [background](https://www.linkedin.com/in/rebecca-rumbul-96a5441a/)
```
